src/main/java/org/broadinstitute/hellbender/tools/exome/CopyNumberTriStateSegment.java

```diff
@@ -0,0 +1,125 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.tools.exome.hmm.CopyNumberTriStateHiddenMarkovModel;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.hmm.CopyNumberTriState;
+
+import java.util.List;
+
+/**
+ * {@link CopyNumberTriStateHiddenMarkovModel} model target segments.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+public final class CopyNumberTriStateSegment extends Segment<CopyNumberTriState> {
+
+    private final double exactQuality;
+
+    private final double someQuality;
+
+    private final double startQuality;
+
+    private final double endQuality;
+
+    private final double eventQuality;
+
+    private final double stdev;
+
+    /**
+     * Returns the "exact quality" (EQ)
+     * @return any {@code double} value.
+     */
+    public double getExactQuality() {
+        return exactQuality;
+    }
+
+    /**
+     * Returns the "some quality" (SQ)
+     * @return any {@code double} value.
+     */
+    public double getSomeQuality() {
```

Comment:
docs: phred-scaled probability that some copy-number event of the same type overlaps this segment.